ecosystem/sep-0012.md

```diff
@@ -48,6 +49,82 @@ Alternatively, if the client cannot add the authorization header. The JWT should
 ?jwt=<token>
 ```
 
+## Customer GET
+
+Checks the status of a customers information. This allows clients to check whether the customers information was accepted, rejected, or still needs more info.  Lookups are done via the stellar account ID and memo that were used for the initial PUT request.
+
+This also gives you the information needed as far as what information a KYC provider requires to KYC a user.  If you fetch an account that has never been KYC'd the server should respond with `NEEDS_INFO` and a list of all the fields you will need to provide.
```

Comment:
I'd personally remove mentions of "you". 

`This endpoint also describes the KYC information the anchor requires of a user.`